### PR TITLE
Enhancement : Support for aggregation on term names

### PR DESCRIPTION
**Pull request by RexterR**
## Change description

> Description here

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mehtaanshul) [inline .vscode/settings.json:1]: Delete this file

File: Atlas/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mehtaanshul) [inline Atlas/.vscode/settings.json:1]: Delete this file

File: Atlas/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## Getting Started
```
Comment (mehtaanshul) [inline Atlas/README.md:1]: Delete this file

File: Atlas/src/App.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+public class App {
```
Comment (mehtaanshul) [inline Atlas/src/App.java:1]: Delete this file

File: distro/bin/.gitignore [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (mehtaanshul) [inline distro/bin/.gitignore:1]: Delete distro/bin folder

File: pom.xml [hunk 6]
```diff
@@ -732,7 +732,7 @@
         <junit.version>4.13.1</junit.version>
         <kafka.scala.binary.version>2.12</kafka.scala.binary.version>
         <kafka.version>2.8.1</kafka.version>
-        <keycloak.version>15.0.2.1</keycloak.version>
+        <keycloak.version>15.0.2</keycloak.version>
```
Comment (mehtaanshul) [inline pom.xml:735]: Remove this change as well, keep it on your local if required

Comment (RexterR) [reply pom.xml]: Done

**Pull request closed without merging**